### PR TITLE
Feat: Add contributions and pr guides

### LINKED ISSUE #1: Add Contribution guidelines

### PR DESCRIPTION
**Pull request by Klin-Coders**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed (if applicable). 

## Related Issue

- Fixes #1 and #17

## Contribution Guidelines

Before submitting this pull request, please review our [Contribution Guidelines](https://github.com/abeg-help/frontend/blob/dev/CONTRIBUTING.md) to understand how to contribute to this project.

## Checklist

- [x] I have reviewed the Contribution Guidelines linked above.
- [x] I have tested my changes thoroughly and ensured that all existing tests pass.
- [x] I have provided clear and concise commit messages.
- [x] I have updated the project's documentation as necessary.

### CODE REVIEW

**Comment by Klin-Coders** [comment]
@babblebey @289Volts @Ryan-Zayne @Toyin5 
check this out and kindly review please

**Comment by 289Volts** [comment]
I love this template. It's straightforward. However, I think Checklist should come last and also a brief explanation of the information that is required for each section so people know what to fill in

**Comment by Toyin5** [comment]
Why don't you include example of how to close an issue?
Like _resolves #issue_number_ or _fixes_ or _closes_ according to the action
That should be in the PR template
@Klin-Coders

**Comment by Klin-Coders** [comment]
> Why don't you include example of how to close an issue? Like _resolves #issue_number_ or _fixes_ or _closes_ according to the action That should be in the PR template @Klin-Coders

i included this part 
```Related Issue
Fixes https://github.com/abeg-help/frontend/issues/1 and https://github.com/abeg-help/frontend/issues/17
```

**Comment by Toyin5** [comment]
Great!
Seems fine then

**Review by 289Volts — CHANGES_REQUESTED** [review]
I love this template. It's straightforward. However, I think Checklist should come last and also a brief explanation of the information that is required for each section so people know what to fill in

**Comment by Klin-Coders** [comment]
> I love this template. It's straightforward. However, I think Checklist should come last and also a brief explanation of the information that is required for each section so people know what to fill in

I think a maintainer should be able to see at first glance if important parts of the checklist are completed, instead of having to scroll through the plethora of screenshots and additional contexts that might be added

also each section is very descriptive, can you point out which part needs extra explanation that the contribution guidlenes does not cover?

they are expected to have read the contribution guidelines before creating a PR as the template suggests

**Review by 289Volts — APPROVED** [review]
Just saw the comments you made in the description. It wasn't showing so i missed it

File: CONTRIBUTING.md
Comment (poriacodes) [inline CONTRIBUTING.md:1]: In the "Coding Standard" section, is there a document that explains our coding standards?

Comment (poriacodes) [inline CONTRIBUTING.md:1]: In the documentation section, let us include:
"To update or create documentation, you need to have Markdown knowledge. 
Visit [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to read about GitHub Markdown and [here](https://www.markdowntutorial.com/) to practice."

Comment (Klin-Coders) [reply CONTRIBUTING.md:1]: yes we have a ```.eslintrc.json``` file in the code template that enforces the code style and lints it. but i added that to ensure people are aware to not forcefully bypass those and ensure their code=ing style is consistent with everyone else.

Comment (Klin-Coders) [reply CONTRIBUTING.md:1]: Okay will do

**Comment by Ryan-Zayne** [comment]
I love the template so far, simple and straightforward as can be

**Pull request merged**